docs_src/getting-started/Ch-GettingStartedSDK-C.md

```diff
@@ -1,213 +1,210 @@
-random number generating Device Service.
-
-1.  Explore the files in the src/c/examples/res folder. Take note of the
-    example Device Profile YAML file that is already there
-    (TemplateProfile.yaml). You can explore the contents of this file to
-    see how devices are represented by YAML. In particular, note how
-    fields or properties of a sensor are represented by
-    "deviceResources". Commands to be issued to the device are
-    represented by "coreCommands".
-2.  Download this
-    `random-generator-device.yaml <random-generator-device.yaml>`{.interpreted-text
-    role="download"} into the ./res folder.
-
-You can open random-generator-device.yaml in a text editor. In this
-Device Profile, you are suggesting that the device you are describing to
-EdgeX has a single property (or deviceResource) which EdgeX should know
-about - in this case, the property is the "randomnumber". Note how the
-deviceResource is typed.
-
-> In more real world IoT situations, this deviceResource list could be
-> extensive and could be filled with all different types of data.
->
-> Note also how the Device Profile describes REST commands that can be
-> used by others to call on (or "get") the random number from the Device
-> Service.
+A device profile is a YAML file that describes a class of device to
+EdgeX. General characteristics about the type of device, the data these devices provide, and how to command the device are all in a device profile.   The device profile tells the device service what data gets collected from the the device and how to get it. 
+
+Follow these steps to create a device profile for the simple random number generating device service.
+
+1.  Explore the files in the device-sdk-c/src/c/examples/res folder.   Note the example TemplateProfile.yaml device profile that is already in this folder.  Open the file with your favorite editor and explore its contents.  Note how `deviceResources` in the file represent properties of a device (properties like SensorOne, SensorTwo and Switch).  Similarly, `coreCommands` specify commands that get issued to the device.
+
+    ![image](EdgeX_SampleDeviceProfile_DeviceResourcesC.png)
+
+2.  A pre-created device profile for the random number device is provided in this documentation.  Download **[random-generator-device.yaml](random-generator-device.yaml)** and save the file to the `./res` folder.
+
+3.  Open the random-generator-device.yaml file in a text editor. In this device profile, the device described has a deviceResource:  `randomnuber`.  Note how the association of a type to the deviceResource.  In this case, the device profile informs EdgeX that randomnumber will be a INT32.  In real world IoT situations, this deviceResource list could be extensive and filled with many deviceResources all different types of data.  Note also how the device profile describes a REST command (GET Random) to call to get the random number from the device service.
```

Comment:
randomnuber should be `randomnumber`. Also the second occurence of this word is not code highlighted like the first occurence.